src/Symfony/Bridge/Doctrine/DependencyInjection/AbstractDoctrineExtension.php

```diff
@@ -384,9 +384,16 @@ protected function loadCacheDriver($cacheName, $objectManagerName, array $cacheD
 
         if (!isset($cacheDriver['namespace'])) {
             // generate a unique namespace for the given application
-            $env = $container->getParameter('kernel.root_dir').$container->getParameter('kernel.environment');
-            $hash = hash('sha256', $env);
-            $namespace = 'sf2'.$this->getMappingResourceExtension().'_'.$objectManagerName.'_'.$hash;
+            if ($container->hasParameter('cache.prefix.seed')) {
+                $seed = '.'.$container->getParameterBag()->resolveValue($container->getParameter('cache.prefix.seed'));
+            } elseif ($container->getParameter('kernel.debug')) {
+                $seed = '_'.$container->getParameter('kernel.root_dir');
```

Comment:
I am not convinced that this is a good idea as it will force developers you deploy more than one application to the same server that share the same cache backend to always configure the `cache.prefix.seed` option.